docs/src/components/productDesignKit/DesignKitValues.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import Box from '@material-ui/core/Box';
+import Container from '@material-ui/core/Container';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import GradientText from 'docs/src/components/typography/GradientText';
+import SvgPalette from 'docs/src/icons/SvgPalette';
```

Comment:
Do we really need new icons or can we re-use the existing `@material-ui/icons` ones? I don't think it's worth adding new icons for some miniscule changes.

Reply:
My understanding is that the new icons will make the rebranding stands out from material design. In some places we also use icons from `@material-ui/icons`. You can take a look in figma to see if the icon make sense or not. 

cc @danilo-leal we should use existing icons as much as possible.

Reply:
These are all custom icons for the website. Does it hurt too much having these customs rather than with the oldies from MD?

Reply:
> Does it hurt too much having these customs rather than with the oldies from MD?

Yep, it adds maintenance burden to the repo. The question is, what is gained by using custom icons. This PR should include minimal changes and later propose new icons so that we can review if we actually need new code.

Reply:
Just to clarify that these icons are not meant to be another complete set or anything like that. They're only custom ones made for the new branding material. The value is in the careful design detail of having it customized. But I get it that adds a lot to the library... We can try to iterate using the MD ones and see how it feels.

Reply:
all icons related to these 2 pages are changed to material design icons.